pra_request_tracker/pra_request_tracker/templates/base.html

```diff
@@ -68,25 +68,6 @@
             <li class="nav-item">
               <a class="nav-link" href="{% url 'record_requests:agencies' %}">Agencies</a>
             </li>
-            {% if request.user.is_authenticated %}
```

Comment:
Why was this removed?

Reply:
We didn't really have anything for the front-end for logged in users, just the admin panel right?